Homework 17

BalaTestPro opened the pull request:
Completed Homework-17

BalaTestPro committed:
Added Registration Navigation

Added a new test case for Registration Navigation.

BalaTestPro committed:
Completed Homework-17

Completed Homework-17

Berichicko commented:
The code is not compiled! Please finish the homework!You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/e95e86a9-90cb-4f14-b958-ba12d112f242)

HINT

**Step 1**

**How to analyze the error:**

Problems tool window (File Project Errors):
The Current File tab shows the list of problems in the file that is open in the editor. Select a problem and press Enter or double-click to navigate to it. You can also open a preview right in the tool window, and you can use the eye icon to filter the problems by type and to change the sorting.

You can select any problem in the window and press F4  or double-click it to jump to the corresponding line in the editor.

For example 
![image](https://github.com/Test-Pro-Bootcamp/QA-Automation-Framework-Template/assets/104761482/7caf54f1-0e32-4164-8d74-ca228119f97f)

The light bulb icon on the toolbar turns on if there are quick fixes for the selected problem. You can also press Alt+Enter on a problem to see the list of available quick fixes.

![image](https://github.com/Test-Pro-Bootcamp/QA-Automation-Framework-Template/assets/104761482/dd115f9c-3459-4476-9c70-2573ad220013)

![image](https://github.com/Test-Pro-Bootcamp/QA-Automation-Framework-Template/assets/104761482/450f92f3-1480-47e4-a951-25438bd30e49)

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/e556f0ed-e934-4c91-971e-1835e3465c45)

**Step 2**

Delete `import java.util.random.RandomGenerator;`

![image](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/5ef0e58f-5659-45ff-a339-dab7cc003d7d)

**RESULT:**

![image](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/bb583ee9-b38c-4357-8a3d-b0bf482b3b83)

**My recommendations:**
Adding multiple arguments:
`--disable-notifications`: disabling notifications
`--remote-allow-origins=*`: allowing remote origins
`--incognito`: launching in incognito mode
`--start-maximized`: launching in maximized mode
`"excludeSwitches", new String[]{"enable-automation"} `: Disable notification Chrome is being controlled by automated test software in ChromeDriver
**These settings are enabled to simplify and improve the process of automated testing.**
_For example_
![image](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/b05c1900-365a-491c-a399-92b5b3bdc666)

BalaTestPro commented:
I'm confused with the comments. I'm able to compile and execute the same code successfully without any issues. The test passed for me. I'm unable to paste the screenshot here.

BalaTestPro committed:
Updated Homework17 based on comments

I've misunderstood the instructions. The Homework17 has been updated now as per the comments. Please review and grade.
